src/content/getting-started/quickstart/k3s/_index.md

```diff
@@ -0,0 +1,133 @@
+---
+date: 2021-02-24T13:36:18+01:00
+title: "K3s"
+weight: 15
+---
+
+This quickstart guide covers the necessary steps to deploy two [K3s](https://k3s.io/) clusters
+and deploy Submariner with Service Discovery to interconnect the two clusters. Note that this
+guide focuses on Submariner deployment on clusters with non-overlapping Pod and Service CIDRs.
+For connecting clusters with overlapping CIDRs, please refer to the
+[Globalnet guide](../../architecture/globalnet/).
+
+### Prerequisites
+
+1. Prepare two nodes with
+[K3s-supported OS](https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#operating-systems)
+installed (These two nodes are referred to as node-a and node-b).
+2. Choose the Pod CIDR and the Service CIDR for each K3s cluster.
+
+    In this guide, we will use the following CIDRs:
+
+    | Cluster   | Pod CIDR     | Service CIDR |
+    |-----------|--------------|--------------|
+    | cluster-a |10.44.0.0/16  |10.45.0.0/16  |
+    | cluster-b |10.144.0.0/16 |10.145.0.0/16 |
+
+### Deploy K3s Clusters
+
+#### Deploy cluster-a on node-a
+
+In this step you will deploy **cluster-a** on node-a using the the specific CIDRs.
+To deploy K3s with the CIDRs, run the following commands on node-a.
+
+```bash
+POD_CIDR=10.44.0.0/16
+SERVICE_CIDR=10.45.0.0/16
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="--cluster-cidr $POD_CIDR --service-cidr $SERVICE_CIDR" sh -s -
+```
+
+{{% notice tip %}}
+For more information on interacting with K3s, please refer to the [k3s documentation](https://rancher.com/docs/k3s/latest/en/quick-start/).
+{{% /notice %}}
+
+The kubeconfig file needs to be modified to set the Kubernetes API endpoint to the public IP of the node-a,
+which is obtained from the first field yielded by `hostname -I`.
+
+```bash
+cp /etc/rancher/k3s/k3s.yaml kubeconfig.cluster-a
+IP=$(hostname -I | awk '{print $1}')
+sed -i 's/127.0.0.1/'$IP'/' kubeconfig.cluster-a
+```
+
+#### Deploy cluster-b on node-b
+
+In this step you will deploy **cluster-b** on node-b using the the specific CIDRs.
+To deploy K3s with the CIDRs, run the following commands on node-b.
+
+```bash
+POD_CIDR=10.144.0.0/16
+SERVICE_CIDR=10.145.0.0/16
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="--cluster-cidr $POD_CIDR --service-cidr $SERVICE_CIDR" sh -s -
+```
+
+The kubeconfig file needs to be modified to set the Kubernetes API endpoint to the public IP of the node-b,
+which is obtained from the first field yielded by `hostname -I`.
+
+```bash
+cp /etc/rancher/k3s/k3s.yaml kubeconfig.cluster-b
+IP=$(hostname -I | awk '{print $1}')
+sed -i 's/127.0.0.1/'$IP'/' kubeconfig.cluster-b
+```
+
+Next, copy kubeconfig.cluster-b to node-a.
+
+#### Install `subctl` on node-a
+
+{{< subctl-install >}}
+
+#### Use cluster-a as the Broker
+
+On node-a, run:
+
+```bash
+subctl deploy-broker --kubeconfig kubeconfig.cluster-a --service-discovery
+```
+
+#### Join cluster-a and cluster-b to the Broker
+
+Join cluster-a to the Broker
```

Comment:
I think it's simpler to just make this its own section - avoids repeating "Join cluster-a ...".

```suggestion
#### Join cluster-a to the Broker
```